### PR TITLE
Avoid usage of the Git and Exec protocols

### PR DESCRIPTION
**Pull request by chmeliik**
Cachi2 does not allow those

I've "backed up" the current state of the branch with an updated README as [disallowed-protocols](https://github.com/cachito-testing/cachi2-yarn-berry/tree/disallowed-protocols) in case we want to use it to test the unsupported locator validation

### CODE REVIEW

**Comment by chmeliik** [comment]
PTAL, if this looks good I'll make the same changes to the zero-installs branch

File: README.md [hunk 1]
```diff
@@ -8,16 +8,18 @@ It does not use zero-installs feature - file `.pnp.cjs` and directory `.yarn/cac
 - [x] Semver
 - [ ] Tag
 - [x] Npm alias
-- [x] Git
+- [ ] ~~Git~~
 - [x] GitHub
```
Comment (taylormadore) [inline README.md]: Haven't we dropped GitHub as well?

Comment (chmeliik) [reply README.md]: Good catch, I forgot those are documented as separate protocols (by Yarn as well)

**Review by ben-alkov — APPROVED** [review]
LGTM

**Review by slimreaper35 — APPROVED** [review]

**Pull request merged**